cli/src/main/kotlin/commands/ConfigCommand.kt

```diff
@@ -50,21 +49,25 @@ class ConfigCommand : CliktCommand(name = "config", help = "Show different ORT c
     ).flag()
 
     private val globalOptionsForSubcommands by requireObject<GlobalOptions>()
-    private val renderOptions = ConfigRenderOptions.defaults().setJson(false).setOriginComments(false)
 
-    private fun OrtConfiguration.renderHocon() = toConfig("ort").root().render(renderOptions)
+    private val mapper = YAMLMapper().apply {
+        registerKotlinModule()
+    }
```

Comment:
Can't / shouldn't we just use `org.ossreviewtoolkit.model.yamlMapper`?

Reply:
That's a difficult question to me: On the one hand using the existing `yamlMapper` would be more consistent with other configuration files because it uses snake case for property names. On the other hand I prefer camel case because I like that the names in the configuration file exactly match the names in Kotlin.

Reply:
As I believe we use `yamlMapper` for `.ort.yml` deserialization, and as noted `.ort.yml` and `config.yml` may partly contain the same configuration options, I guess we need to stay consistent and use the same (de-)serializer for both configuration files, so users could copy over part of the global config to the repo config and alter it.

Reply:
Unfortunately I tend to agree. I'd rather reconsider if we should switch all other file to camel case, but that's too big a change for now...
I need to check if hoplite-yaml can be configured to deserialize snake case, but I would expect that this is possible.

Reply:
@sschuberth and me discussed this offline and came to the conclusion that the file will continue using camel case, because there are some places where it cannot use snake case, for example in the scanner options map where the keys need to match the class names exactly. Instead, we would indeed prefer to convert the other files from snake case to camel case, I will create an issue for that with the reasoning.